app/controllers/articles_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ArticlesController < ApplicationController
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
+      flash[:success] = 'Article successfully created'
+      redirect_to article_path(@article)
+    else
+      if params[:title].blank? || params[:body].blank?
```

Comment:
This one is a little bit problematic for me.  Please use the error message active record provides for us if article is not being saved.

Reply:
Will have to look in to this. But we can make that happen.